LABS-EU3/hackton-frontend
# Chore/setup redux

#### What does this PR do?
Setup configuration for redux

#### Description of Task to be completed?
- Document the code with annotations
- Writing unit tests

#### How should this be manually tested?
Checkout to `chore/configure-redux` branch
Run `npm install`
Check `src/store` for the configurations 

#### Any background context you want to provide?



#### Screenshots (if appropriate)

#### Questions:
